%% BioMed_Central_Tex_Template_v1.06
%%                                      %
%  bmc_article.tex            ver: 1.06 %
%                                       %

%%IMPORTANT: do not delete the first line of this template
%%It must be present to enable the BMC Submission system to
%%recognise this template!!

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                     %%
%%  LaTeX template for BioMed Central  %%
%%     journal article submissions     %%
%%                                     %%
%%          <8 June 2012>              %%
%%                                     %%
%%                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% For instructions on how to fill out this Tex template           %%
%% document please refer to Readme.html and the instructions for   %%
%% authors page on the biomed central website                      %%
%% http://www.biomedcentral.com/info/authors/                      %%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%% BioMed Central currently use the MikTex distribution of         %%
%% TeX for Windows) of TeX and LaTeX.  This is available from      %%
%% http://www.miktex.org                                           %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% additional documentclass options:
%  [doublespacing]
%  [linenumbers]   - put the line numbers on margins

%%% loading packages, author definitions

%\documentclass[twocolumn]{bmcart}% uncomment this for twocolumn layout and comment line below
\documentclass{bmcart}

%%% Load packages
\usepackage{amsthm,amsmath}
\RequirePackage{natbib}
\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\def\includegraphic{}
\def\includegraphics{}



%%% Put your definitions there:
\startlocaldefs
\setlength{\marginparwidth}{2cm}
\usepackage{url}
%TODO remove graphicx
%\usepackage{graphicx}
\endlocaldefs


%%% Begin ...
\begin{document}

%%% Start of article front matter
\begin{frontmatter}

\begin{fmbox}
\dochead{Data Note}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title{Deeply sequenced metagenome and metatranscriptome of a biogas-producing microbial community from an agricultural production-scale biogas plant}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\author[
   addressref={cebitec,techfak},
   corref={cebitec},
   email={abremges@cebitec.uni-bielefeld.de}
]{\inits{AB}\fnm{Andreas} \snm{Bremges}}
\author[
   addressref={cebitec},
   email={irena.maus@cebitec.uni-bielefeld.de}
]{\inits{IM}\fnm{Irena} \snm{Maus}}
\author[
   addressref={cebitec,techfak},
   email={pbelmann@cebitec.uni-bielefeld.de}
]{\inits{PB}\fnm{Peter} \snm{Belmann}}
\author[
   addressref={cebitec},
   email={feikmeye@cebitec.uni-bielefeld.de}
]{\inits{FE}\fnm{Felix} \snm{Eikmeyer}}
\author[
   addressref={cebitec},
   email={awinkler@cebitec.uni-bielefeld.de}
]{\inits{AW}\fnm{Anika} \snm{Winkler}}
\author[
   addressref={cebitec},
   email={aalbersm@cebitec.uni-bielefeld.de}
]{\inits{AA}\fnm{Andreas} \snm{Albersmeier}}
\author[
   addressref={cebitec},
   email={puehler@cebitec.uni-bielefeld.de}
]{\inits{AP}\fnm{Alfred} \snm{Pühler}}
\author[
   addressref={cebitec},
   noteref={n1},
   email={aschluet@cebitec.uni-bielefeld.de}
]{\inits{ASch}\fnm{Andreas} \snm{Schlüter}}
\author[
   addressref={cebitec,techfak},
%   corref={cebitec},
   noteref={n1},
   email={asczyrba@cebitec.uni-bielefeld.de}
]{\inits{AScz}\fnm{Alexander} \snm{Sczyrba}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\address[id=cebitec]{%                           % unique id
  \orgname{Center for Biotechnology, Bielefeld University}, % university, etc
  %\street{Universitätsstraße 27},                     %
  \postcode{33615}                                % post or zip code
  \city{Bielefeld},                              % city
  \cny{Germany}                                    % country
}
\address[id=techfak]{%
  \orgname{Faculty of Technology, Bielefeld University},
  %\street{Universitätsstraße 25},
  \postcode{33615}
  \city{Bielefeld},
  \cny{Germany}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{artnotes}
%\note{Sample of title note}     % note to the article
\note[id=n1]{Equal contributor} % note, connected to author
\end{artnotes}

\end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Abstract begins here                 %%
%%                                          %%
%% Please refer to the Instructions for     %%
%% authors on http://www.biomedcentral.com  %%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstractbox}

\begin{abstract} % abstract
\parttitle{Background}
The production of biogas takes place under anaerobic conditions and involves microbial decomposition of organic matter. Most of the participating microbes still have to be considered as unknown and non-cultivable. Accordingly, shotgun metagenome sequencing currently is the method of choice to obtain insights into community composition and the genetic repertoire.
%a presentation of the interest or relevance of these data for the broader community
%
\parttitle{Findings}
Here, we report the deeply sequenced metagenome and metatranscriptome of a complex biogas-producing microbial community from an agricultural production-scale biogas plant. We assembled the metagenome and e.g. reconstructed most genes involved in the methane metabolism, a key pathway involving methanogenesis performed by methanogenic \emph{Archaea}. This exemplary result indicates sufficient sequencing coverage for most downstream analyses.
%a very brief preview of the data type(s) produced, the methods used, and information relevant to data validation
%
\parttitle{Conclusions}
Sequenced at least one order of magnitude deeper than previous studies, our metagenome data will enable novel insights into community composition and the genetic potential of important community members. Moreover, mapping of transcripts to reconstructed genome sequences will enable the identification of active metabolic pathways in target organisms.
%a short summary of the potential uses of these data and implications for the field
\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{keyword}
\kwd{Biogas}
\kwd{Anaerobic digestion}
\kwd{Wet fermentation}
\kwd{Methanogenesis}
\kwd{Metagenomics}
\kwd{Metatranscriptomics}
\kwd{Sequencing}
\kwd{Assembly}
\end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

\end{abstractbox}
%
%\end{fmbox}% uncomment this for twcolumn layout

\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Main Body begins here                %%
%%                                          %%
%% Please refer to the instructions for     %%
%% authors on:                              %%
%% http://www.biomedcentral.com/info/authors%%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%% See the Results and Discussion section   %%
%% for details on how to create sub-sections%%
%%                                          %%
%% use \cite{...} to cite references        %%
%%  \cite{koon} and                         %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}    %%
%%  \nocite{smith,marg,hunn,advi,koha,mouse}%%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

\section*{Data description}

\subsection*{Background}
Production of biogas by means of anaerobic digestion of biomass is becoming increasingly important as biogas is regarded a clean, renewable and environmentally compatible energy source \cite{Weiland2010}. Moreover, generation of energy from biogas relies on a balanced carbon dioxide cycle.

The process of biogas production takes place under anaerobic conditions and involves microbial decomposition of organic matter, yielding methane as the main final product of the fermentation process. Complex consortia of microorganisms are responsible for biomass decomposition and biogas production.
The majority of the participating microbes are still unknown, as is their influence on reactor performance. Since most of the organisms within biogas communities are non-cultivable by today’s conventional microbiological techniques, sequencing of metagenomic total community DNA currently is the best way to obtain unbiased insights into community composition and the metabolic potential of key community members.

Here, we describe the deeply sequenced metagenome and metatranscriptome of an agricultural production-scale biogas plant on the Illumina platform \cite{GigaDB}.
We sequenced the metagenome $27 \times$ and $19 \times$ deeper, respectively, than previous studies applying 454 or SOLiD sequencing \cite{Jaenicke2011, Wirth2012}, primarily focusing on community composition.
%454 \cite{Jaenicke2011}: 1,963,716 reads, 843,091,863 bases: factor of 27.3 more
%SOLiD \cite{Wirth2012}: 23,897,590 reads, 1,194,879,500 bases: factor of 19.3 more
%No idea about the 454 data from Solli, 2014: http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA261310
Metatranscriptomic sequencing of total community RNA, $230 \times$ deeper than previously reported \cite{Zakrzewski2012}, complements the metagenome.
%454 \cite{Zakrzewski2012}: 484,920 reads, 55,164,919 bases: factor of 229.8 more
Combined, these data will enable a deeper exploration of the biogas-producing microbial community, with the objective to develop rational strategies for process optimization.


\subsection*{Digester management and process characterization}
The biogas plant, located in North Rhine Westphalia, Germany, features a mesophilic continuous wet fermentation technology characterized recently \cite{Stolze2015}.
It was designed for a capacity of $537\,kW_{el}$ combined heat and power (CHP) generation.
The process comprises three digesters: a primary and secondary digester, where the main proportion of biogas is produced, and a storage tank, where the digestate is fermented thereafter.

The primary digester is fed hourly with a mixture of $72\,\%$ maize silage and $28\,\%$ liquid pig manure.
The biogas and methane yields at the time of sampling were at $810.5$ and $417.8$ liters per kg organic dry matter ($l / kg\,oDM$), respectively.
After a theoretical retention time of $55$ days, the digestate is stored in the closed, non-heated final storage tank.
Further metadata are summarized in Table \ref{tBiogasPlant}.

\subsection*{Sampling and library construction}
Samples from the primary digester of the aforementioned biogas plant were taken in November 2010.
Prior to the sampling process, approximately $15\,l$ of the fermenter substrate were discarded before aliquots of $1\,l$ were transferred into clean gastight sampling vessels and transported directly to the laboratory.

For the metagenome, aliquots of $20\,g$ of the fermentation sample were used for total community DNA preparation as described previously \cite{Schlueter2008}.

For the metatranscriptome, a random-primed cDNA library was prepared by an external vendor (Vertis Biotechnologie AG). Briefly, total RNA was first treated with $5'$-P dependent Terminator exonuclease (Epicentre) to enrich for full-length mRNA carrying $5'$ CAP or triphosphate structures. Then, first-strand cDNA was synthesized using a N6 random primer and M-MLV-RNase H reverse transcriptase, and second-strand cDNA synthesis was performed according to the Gubler-Hoffman protocol \cite{GublerHoffman}.

\subsection*{Metagenomic and metatranscriptomic sequencing}
We sequenced one metatranscriptome and two metagenome shotgun libraries on Illumina's Genome Analyzer IIx system, applying the Paired-End DNA Sample Preparation Kit (Illumina Inc.) as described by the manufacturer and generating $2 \times 161\,bp$ paired-end reads.
On Illumina's MiSeq system, we sequenced three further metagenome shotgun libraries, applying the Nextera DNA Sample Preparation Kit (Illumina Inc.) as described by the manufacturer and generating $2 \times 155\,bp$ paired-end reads.
Our sequencing efforts, yielding $35$ gigabases in total, are summarized in Table \ref{tReads}.

\subsection*{Metagenome assembly}
Prior to assembly, we used Trimmomatic \cite{Trimmomatic}, version 0.33, for adapter removal and moderate quality trimming.
After adapter clipping, using Trimmomatic's \emph{Truseq2-PE} and \emph{Nextera-PE} templates, we removed leading and trailing ambiguous or low quality bases (below Phred quality scores of 3).
Table \ref{tPostQC} summarizes the effect on sequencing depth, more than $25$ gigabases of sequence data passed quality control.

We assembled the metagenome with Ray Meta \cite{RayMeta}, version 2.3.1, trying a range of $k$-mer sizes from $21$ to $61$ in steps of $10$.
To estimate the inclusivity of the set of assemblies, we aligned the post-QC sequencing reads to the assembled contigs with bowtie2 \cite{Bowtie2}, version 2.2.4.
We then used samtools \cite{Samtools}, version 1.1, to convert SAM to BAM, sort the alignment file, and calculate the mapping statistics.
Based on total assembly size, contiguity, and the percentage of mapped back metagenomic reads, we selected the assembly produced with a $k$-mer size of $31$.
Here, we assembled approximately $228$ megabases in $54,489$ contigs greater than $1,000\,bp$, with an N50 value of $9,796\,bp$. % It's over 9000!
$77\,\%$ ($79\,\%$) of metagenomic (metatranscriptomic) reads mapped back to this assembly.

\subsection*{Gene prediction and annotation}
We used MetaProdigal \cite{MetaProdigal}, version 2.6.1, to predict $250,596$ protein-coding genes on the assembled contigs.
We blasted the protein sequences of all predicted genes against the KEGG database \cite{KeggDB}, release 72.0, using Protein-Protein BLAST \cite{BlastPlus}, version 2.2.29+. 
Of the $250,596$ predicted genes, $191,766$ ($76.5\,\%$) had a match in the KEGG database, using an Evalue cutoff of $10^{-6}$.
We determined the KEGG Orthology (KO) for each gene by mapping the top-scoring BLAST hit to its orthologous gene in KEGG, resulting in $109,501$ genes with an assigned KEGG Orthology.
Table \ref{tAssembly} summarizes our results.

\subsection*{Relating the metagenome and the metatranscriptome}
To illustrate potential use cases, we first counted the number of reads within genes using BEDTools \cite{BEDTools}, version 2.22.0, and highlighted metagenomic and metatranscriptomic coverage of the methane metabolism pathway in Figure \ref{fPathway}.
We therefore assembled the majority of genes involved in the methane metabolism from our metagenomic data, with accompanying metatranscriptomic data suggesting active gene expression for many.

For a second example, we calculated the reads per kilobase per million mapped reads (RPKM) for each gene as a crude measure for abundance (metagenome) or expression (metatranscriptome).
Figure \ref{fCoverage} relates the two. Here, we accentuated all genes assigned to either of the three known types of methanogenic pathways.
Hydrogenotrophic methanogenesis, i.e. the reduction of CO$_{\text{2}}$ with hydrogen, appears to be highly expressed in the reactor analyzed, which is in agreement with results obtained via 454 amplicon and metatranscriptome sequencing \cite{Zakrzewski2012}.

\section*{Discussion}

We report extensive metagenomic and metatranscriptomic profiling of the microbial community from a production-scale biogas plant.
Given the unprecedented sequencing depth and established bioinformatics, our data are of great interest to the biogas research community in general and microbiologists working on biogas-producing microbial communities in particular.
In a first applied study, our metagenome assembly was used to improve the characterization of a metaproteome generated from biogas plant fermentation samples and to investigate the metabolic activity of the microbial community \cite{Kohrs2015}.

By sharing our data, we want to actively encourage its reuse. This will hopefully result in novel biological and biotechnological insights, eventually enabling a more efficient biogas production.

\section*{Availability}

\subsection*{Data accession}
Raw sequencing data are available in the European Nucleotide Archive (ENA) under study accession \href{http://www.ebi.ac.uk/ena/data/view/PRJEB8813}{PRJEB8813}. The datasets supporting the results of this article are available in \emph{GigaScience}'s GigaDB \cite{GigaDB}.

\subsection*{Reproducibility}
The complete workflow is organized in a single GNU Makefile and available on GitHub \cite{GitHub}.
All data and results can be reproduced by a simple invocation of \emph{make}.
To further support reproducibility, we bundled all tools and dependencies into one Docker container available on DockerHub \cite{DockerHub}. \emph{docker run} executes the aforementioned Makefile inside the container.
Reproduction requires roughly $89\,GiB$ memory and $83\,GiB$ storage, and takes less than $24$ hours on $32$ CPU cores.

Excluding the KEGG analysis, which relies on a commercial license of the KEGG database, all steps are performed using free and open-source software.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Backmatter begins here                   %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{backmatter}

\section*{Competing interests}
The authors declare that they have no competing interests.

\section*{Author's contributions}
AB conveived and performed all bioinformatic analyses and wrote the paper.
IM investigated all metadata and drafted part of the data description.
PB implemented the accompanying Docker container.
FE collected the study material.
AW and AA provided the sequencing service.
AP acquired funding and revised the paper.
ASch and AScz jointly directed the project and extensively revised the paper.
All authors read and approved the final manuscript.

\section*{Acknowledgements}
AB, IM, and FE are supported by a fellowship from the CLIB Graduate Cluster Industrial Biotechnology.
AScz is supported by an AWS in Education Research Grant award.
We gratefully acknowledge funding by the German Federal Ministry of Food and Agriculture (BMEL), grant number 22006712 (joint research project Biogas-Core) and the German Federal Ministry of Education and Research (BMBF), grant number 03SF0440C (joint research project Biogas-Marker).
We acknowledge support of the publication fee by Deutsche Forschungsgemeinschaft and the Open Access Publication Funds of Bielefeld University.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                  The Bibliography                       %%
%%                                                         %%
%%  Bmc_mathpys.bst  will be used to                       %%
%%  create a .BBL file for submission.                     %%
%%  After submission of the .TEX file,                     %%
%%  you will be prompted to submit your .BBL file.         %%
%%                                                         %%
%%                                                         %%
%%  Note that the displayed Bibliography will not          %%
%%  necessarily be rendered by Latex exactly as specified  %%
%%  in the online Instructions for Authors.                %%
%%                                                         %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% if your bibliography is in bibtex format, use those commands:
\bibliographystyle{bmc-mathphys} % Style BST file
\bibliography{bremges_gigascience_2015}      % Bibliography file (usually '*.bib' )
% or include bibliography directly:
% \begin{thebibliography}
% \bibitem{b1}
% \end{thebibliography}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Figures                       %%
%%                               %%
%% NB: this is for captions and  %%
%% Titles. All graphics must be  %%
%% submitted separately and NOT  %%
%% included in the Tex document  %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%
%% Do not use \listoffigures as most will included as separate files

% TODO remove newpage
%\newpage
\section*{Figures}
\begin{figure}[h!]
\centering
%\includegraphics[width=.9\textwidth]{Fig1.png}
\caption{\csentence{Methane metabolism pathway analysis.}
Genes reconstructed in our assembly, that are involved in the methane metabolism [PATH:\href{http://www.genome.jp/kegg-bin/show_pathway?ko00680}{ko00680}], are highlighted: Genes with only metagenomic support in yellow, genes with also metatranscriptomic support, suggesting active gene expression, in orange.
Methane is synthesized from CO$_{\text{2}}$, methanol, or acetate.
KEGG pathway map courtesy of Kanehisa Laboratories.}
\label{fPathway}
\end{figure}
\begin{figure}[h!]
\centering
%\includegraphics[width=.9\textwidth]{Fig2.pdf}
\caption{\csentence{Relating the metagenome and metatranscriptome.}
Highlighted are genes involved in methanogenesis, color coded by pathway type: CO$_{\text{2}}$ to methane [MD:\href{http://www.kegg.jp/kegg-bin/show_module?M00567}{M00567}] green ($96$ genes), methanol to methane [MD:\href{http://www.kegg.jp/kegg-bin/show_module?M00356}{M00356}] red ($5$ genes), and acetate to methane [MD:\href{http://www.kegg.jp/kegg-bin/show_module?M00357}{M00357}] blue ($209$ genes).
Common genes, shared between pathway types, are yellow ($80$ genes).
In the background a two-dimensional density estimation for all $250,596$ genes.}
\label{fCoverage}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Tables                        %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% Use of \listoftables is discouraged.
%%

% TODO remove newpage
%\newpage
\section*{Tables}
\begin{table}[h!]
\caption{Characteristics of the studied biogas plant. Primary digester, sampling date: Nov 15, 2010.}
\begin{tabular}{rl}
\hline
Process parameter & Sample\\
\hline
Net volume & $2,041\,m^{3}$\\
Dimensions & $6.4\,m$ high, diameter of $21\,m$\\
Electrical capacity & $537\,kW_{el}$\\
\hline
pH & $7.83$\\
Temperature & $40\,^{\circ}{\rm C}$\\
Conductivity & $22.10\,mS/cm$\\
Volative organic acids (VOA) & $5,327\,mg/l$\\
Total inorganic carbon (TIC) & $14,397\,mg/l$\\
VOA/TIC & $0.37$\\
Ammoniacal nitrogen & $2.93\,g/l$\\
Acetic acid & $863\,mg/l$\\
Propionic acid & $76\,mg/l$\\
\hline
Fed substrates & $72\,\%$ maize silage, $28\,\%$ pig manure\\
Organic load & $4.0\,kg\,oDM\,m^{-3}\,d^{-1}$\\
Retention time & $55\,d$\\
Biogas yield & $810.5\,l/kg\,oDM$\\
Methane yield & $417.8\,l/kg\,oDM$\\
\hline
\end{tabular}
\label{tBiogasPlant}
\end{table}

\begin{table}[h!]
\caption{Overview of the different sequencing libraries.}
\begin{tabular}{lllrrrr}
\hline
Accession & Library name & Library type & Insert size$^{1}$ & Cycles & Reads & Bases \\
\hline
ERS697694 & GAIIx, Lane 6 & RNA, TruSeq & $202 \pm 49$ & $2 \times 161$ & $78,752,308$ & $12,679,121,588$ \\
ERS697688 & GAIIx, Lane 7 & DNA, TruSeq & $157 \pm 19$ & $2 \times 161$ & $54,630,090$ & $8,795,444,490$ \\
ERS697689 & GAIIx, Lane 8 & DNA, TruSeq & $298 \pm 32$ & $2 \times 161$ & $74,547,252$ & $12,002,107,572$ \\
ERS697690 & MiSeq, Run A1$^{2}$ & DNA, Nextera & $173 \pm 53$ & $2 \times 155$ & $4,915,698$ & $761,933,190$ \\
ERS697691 & MiSeq, Run A2$^{2}$ & DNA, Nextera$^{3}$ & $522 \pm 88$ & $2 \times 155$ & $1,927,244$ & $298,722,820$ \\
ERS697692 & MiSeq, Run B1$^{2}$ & DNA, Nextera & $249 \pm 30$ & $2 \times 155$ & $3,840,850$ & $573,901,713$ \\
ERS697693 & MiSeq, Run B2$^{2}$ & DNA, Nextera$^{3}$ & $525 \pm 90$ & $2 \times 155$ & $4,114,304$ & $614,787,564$ \\
\hline
\end{tabular}
\\{$^{1}$Insert sizes determined with Picard tools. $^{2}$Partial runs. $^{3}$This Nextera library was sequenced twice.}
\label{tReads}
\end{table}

\begin{table}[h!]
\caption{Metagenomic and metatranscriptomic sequencing and quality control.}
\begin{tabular}{lrrrr}
\hline
Library type & Reads, raw & post-QC & Bases, raw & post-QC\\
\hline
Metagenome (total) & $143,975,438$ & $137,365,053$ & $23,046,897,349$ & $17,267,320,221$ \\
Metatranscriptome & $78,752,308$ & $73,165,986$ & $12,679,121,588$ & $8,455,809,264$ \\
\hline
\end{tabular}
\label{tPostQC}
\end{table}

\begin{table}[h!]
\caption{Metagenome assembly statistics, minimum contig size of $1,000\,bp$.}
\begin{tabular}{rl}
\hline
Assembly metric & Our assembly\\
\hline
Total size & $228,382,457\,bp$\\
Number of contigs & $54,489$\\
N50 value & $9,796\,bp$\\
Largest contig & $333,979\,bp$\\
\hline
Mapped DNA reads & $105,461,596\,(77\,\%)$\\
Mapped RNA reads & $57,436,058\,(79\,\%)$\\
\hline
Predicted genes & $250,596$\\
of these, full-length & $172,372\,(69\,\%)$\\
Match in KEGG Genes & $191,766$\\
of these, assigned KO & $109,501$\\
of these, in KEGG pathways & $61,100$\\
\hline
\end{tabular}
\label{tAssembly}
\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Additional Files              %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\section*{Additional Files}
%  \subsection*{Additional file 1 --- Sample additional file title}
%    Additional file descriptions text (including details of how to
%    view the file, if it is in a non-standard format or the file extension).  This might
%    refer to a multi-page table or a figure.
%
%  \subsection*{Additional file 2 --- Sample additional file title}
%    Additional file descriptions text.


\end{backmatter}
\end{document}
